docs/user-guide/projects/project-lists/README.md

```diff
@@ -148,6 +155,46 @@ By selecting *Overall activity* you can open a view in which all the latest glob
 
 ![overall activity meeting filter](actvity-global-filter.png)
 
+## Favorite project lists
+
+You can mark your private project lists as favorites. This includes shared, public and private project lists, but not the static ones, e.g. *Active projects* or *Archived projects*. To mark a project list as favorite click the star icon in the top right corner. 
+
+![Mark project list as favorite in OpenProject](star-project-list.png)
+
+The star will turn yellow and the favorite project list will move to the top of the list within the respective sidebar section. If multiple project lists are favored, they will be listed alphabetically.
+
+![Favorite project list in OpenProject](star-yellow-project-list.png)
+
+You can remove the star by clicking on the star icon again. 
+
+## Share project lists 
+
+You can share project lists in OpenProject, except the static project lists, e.g. *Active projects* or project lists under *Project status* section in the lefthand menu. Project lists can either be made public, i.e. shared with everyone at OpenProject or with specific users or groups.
+
+### Share with everyone at OpenProject
+
+You can share a project list with everyone at OpenProject, thus making it public. This means that the project list will become visible to all the users registered on your OpenProject instance. 
+
+To do that click the **Share** icon and turn on the **Share with everyone at OpenProject** switch.
+
+![Share a project list with everyone in OpenProject](project-lists-share-with-everyone-button.png)
+It will be added to the menu on the left under *Public project lists*.
+
+![Public project lists in OpenProject](public-project-lists.png)
+
+### Share project lists with users and groups (Enterprise add-on)
+> [!NOTE] 
+> Sharing project lists with users and groups is an Enterprise add-on and will only be displayed here for Enterprise on-premises and Enterprise cloud.
+
+You can share private project lists with specific users or groups in OpenProject. To do that navigate to a project list and click the **Share** icon. Then specify a user or a group using the search field, define whether they can only view or edit a project list and finally click the **Share** button. That user or group will see that shared project list under **Shared project lists**.  
+
+If a project list has already been shared, you will see the list of users that have access to the project list, including the list owner. You will can modify or revoke project list sharing privileges at any time.
+
+![Share project lists with users or groups in OpenProject](projects-list-share-enterprise.png)
+
+> [!NOTE]
+> Individual project permissions will override project lists. That means that within the project lists users will only see the projects that they already have access to.
```

Comment:
@MayaBerd the fact of which projects a user can see and which no its related with access to that project not to the permissions of that user on that project. Therefore, I would change the sentence to what we used in the info banner inside the modal "Users will only see the projects they have access to. Sharing project lists does not impact individual project permissions." or something similar.